README.md

```diff
@@ -1,50 +1,46 @@
 
-yaml-cpp uses [CMake](http://www.cmake.org) to support cross-platform building. The basic steps to build are:
+## How to Build
 
-1. Download and install [CMake](http://www.cmake.org) (Resources -> Download).
+`yaml-cpp` uses [CMake](http://www.cmake.org) to support cross-platform building. Install [CMake](http://www.cmake.org) _(Resources -> Download)_ before proceeding. The basic steps to build are:
 
-**Note:** If you don't use the provided installer for your platform, make sure that you add CMake's bin folder to your path.
+**Note:** If you don't use the provided installer for your platform, make sure that you add `CMake`'s bin folder to your path.
 
-2. Navigate into the source directory, and type:
+#### 1. Navigate into the source directory, create build folder and run `CMake`:
 
-```
+```sh
 mkdir build
 cd build
+cmake [-G generator] [-DBUILD_SHARED_LIBS=on|OFF] ..
 ```
 
-3. Run CMake. The basic syntax is:
-
-```
-cmake [-G generator] [-DYAML_BUILD_SHARED_LIBS=ON|OFF] ..
-```
-
-  * The `generator` is whatever type of build system you'd like to use. To see a full list of generators on your platform, just run `cmake` (with no arguments). For example:
-    * On Windows, you might use "Visual Studio 12 2013" to generate a Visual Studio 2013 solution or "Visual Studio 14 2015 Win64" to generate a 64-bit Visual Studio 2015 solution.
-    * On OS X, you might use "Xcode" to generate an Xcode project
-    * On a UNIX-y system, simply omit the option to generate a makefile
+  * The `generator` option is the build system you'd like to use. Run `cmake` without arguments to see a full list of available generators.
+    * On Windows, you might use "Visual Studio 12 2013" (VS 2013 32-bits), or "Visual Studio 14 2015 Win64" (VS 2015 64-bits).
+    * On OS X, you might use "Xcode".
+    * On a UNIX-like system, omit the option (for a Makefile).
 
-  * yaml-cpp defaults to building a static library, but you may build a shared library by specifying `-DYAML_BUILD_SHARED_LIBS=ON`.
+  * `yaml-cpp` builds a static library by default, you may want to build a shared library by specifying `-DBUILD_SHARED_LIBS=ON`.
 
   * For more options on customizing the build, see the [CMakeLists.txt](https://github.com/jbeder/yaml-cpp/blob/master/CMakeLists.txt) file.
 
-4. Build it!
+#### 2. Build it!
+  * The command you'll need to run depends on the generator you chose earlier.
 
-5. To clean up, just remove the `build` directory.
+**Note:** To clean up, just remove the `build` directory.
 
-# Recent Release #
+## Recent Releases
 
-[yaml-cpp 0.6.0](https://github.com/jbeder/yaml-cpp/releases/tag/yaml-cpp-0.6.0) has been released! This release requires C++11, and no longer depends on Boost.
+[yaml-cpp 0.6.0](https://github.com/jbeder/yaml-cpp/releases/tag/yaml-cpp-0.6.0) released! This release requires C++11, and no longer depends on _Boost_.
```

Comment:
I personally wouldn't italicize Boost, but I guess not a big deal.

Reply:
Removed